server/views/pages/adjustments/tagged-bail/select-case.njk

```diff
@@ -36,7 +36,14 @@
     {% for activeSentence in model.activeSentences() %}
       <div class="govuk-summary-card">
         <div class="govuk-summary-card__title-wrapper">
-          <h2 class="govuk-summary-card__title">{{ activeSentence.courtDescription }}</h2>
+          {% if activeSentence.recall %}
```

Comment:
put the if around the strong tag, rather than here. Then there is no duplication of the court description.

Reply:
I've updated the code as you've suggested